msteams-platform/TOC.yml

```
@@ -370,6 +370,8 @@
       href: graph-api/rsc/test-rsc-json-file.md
 - name: Publish your app
   items:    
+  - name: Add defaultInstallScope
```

```suggestion
  - name: Configure app's default install scope
```
A TOC label should be more user friendly. How about this?